migrations/2025-02-17-133723_timezones/up.sql

```diff
@@ -0,0 +1,45 @@
+SET timezone = 'UTC';
```

Comment:
this part is important since Postgres would otherwise perform a full table rewrite. if we set the timezone to `UTC`, Postgres can take a shortcut and doesn't have to rewrite any data.

Reply:
Yeah, this is interesting, which I also wanted to point out :)
Here is a SO link[^1] for reference that provides more details about this.

[^1]: https://dba.stackexchange.com/questions/322904/alter-timestamp-column-to-timestamptz-without-converting-data/322907#322907

Reply:
migration started on staging at 2025-02-18T08:28:22.618774Z 
migration ended on staging at 2025-02-18T08:28:22.653333Z

35ms seems reasonable IMHO :)